### ISSUE TITLE
Allow formElements to be inputs with type `number`

### ISSUE THREAD

**Issue by veryrusty** [freqdec/datePicker#53]
Is it possible to include inputs with `type="number"` as allowable fields to bind the date picker too?

Our use-case is HTML5 based content where a date is being collected with the day and year fields as inputs with type `number` for keyboard selection on mobile devices, rather than the default text keyboard.

**Comment by freqdec** [comment]
Yep, no problem, I shall add it to the V7 branch of the datepicker that should be landing in the next few weeks.
